etc/config/cuda.amazon.properties

```diff
@@ -1,7 +1,10 @@
 compilers=&nvcc:&cuclang
 defaultCompiler=nvcc92
-supportsBinary=false
+supportsBinary=true
```

Comment:
I don't think binary works for cuclang, so I would suggest supportsBinary=false should be default, and group.nvcc.supportsBinary=true

Reply:
Fixed.